_build/templates/default/sass/_colors-and-vars.scss

```
@@ -19,8 +19,12 @@ $black: darken($colorSplash, 42.5%); /* generate the black from the $colorSplash
 $yellow: #FFEA80;
 $orange: #F49548;
 $red: #BE0000;
+$lighterRed: #FFEEEE;
 $green: #6CB24A;
+$lighterGreen: #EFFCF6;
+$greener: darken($green, 12%);
 $blue: #3697cd; /* default theme $colorSplash */
+$havelockBlue: #507dd7;
```

```suggestion
$havelockBlue: #507DD7;
```

Improve code quality by using capitalization consistently.

---

Are there editor styles available for this?

---

> Are there editor styles available for this?

@Jako in 137 line

---

I meant by this file: https://github.com/modxcms/revolution/blob/3.x/.editorconfig

---

In PhpStorm these styles could be predefined in https://www.jetbrains.com/help/phpstorm/settings-code-style-scss.html. Maybe there is a way to put those settings in the .editorconfig.

---

Open #14311